### ISSUE TITLE
dns_nameservers not working when set through the config.yaml

### ISSUE THREAD

**Issue by tfiduccia** [rancher/k3os#96]
Version - v0.2.1-rc1

1.  Create a config.yaml with
```
k3os:
  dns_nameservers:
  - 1.1.1.1
  password: asdf
```
2.  During boot process, press e to get into GNU Grub
3.  On linux cmdline add k3os.mode=install k3os.install.config_url="config yaml url"
4.  Log in and cat /etc/resolv.conf

Results:  1.1.1.1 is not in list of namespaces.  I also looked in the /etc/connman/main.conf and it wasn't in there either.

**Comment by axsc** [comment]
I have the same problem with version 0.2.1.
At the moment you can configure your nameserver with connman.

**Comment by pagong** [comment]
Same problem for me: `dns_nameservers` and `ntp_servers` configured in `config.yaml`
```
  dns_nameservers:
  - 192.168.4.1
  - 1.1.1.1
  - 8.8.8.8
  ntp_servers:
  - 192.168.4.1
  - pool.ntp.org
```
are written to a file named '/etc/connman/main.conf ' <- please observe the trailing space !!
```
NetworkInterfaceBlacklist=veth
FallbackNameservers=192.168.4.1,1.1.1.1,8.8.8.8
FallbackTimeservers=192.168.4.1,pool.ntp.org
```
In '/etc/connman' there is another file called 'main.conf' <- without trailing space !

**Comment by axsc** [comment]
Very strange.
I thought that the nameserver and ntp-settings should be in a file under "/var/lib/connman/*"

**Comment by liyimeng** [comment]
@pagong You are likely find a bug, why not make a pull request?

**Comment by zimme** [comment]
This should be solved with the following PR's #137 #149.

**Comment by dweomer** [comment]
> This should be solved with the following PR's #137 #149.
